drivers/gles3/storage/light_storage.cpp

```diff
@@ -46,6 +47,11 @@ LightStorage *LightStorage::get_singleton() {
 
 LightStorage::LightStorage() {
 	singleton = this;
+
+	directional_shadow.size = GLOBAL_GET("rendering/lights_and_shadows/directional_shadow/size");
+	directional_shadow.use_16_bits = GLOBAL_GET("rendering/lights_and_shadows/directional_shadow/16_bits");
+
+	// lightmap_probe_capture_update_speed = GLOBAL_GET("rendering/lightmapping/probe_capture/update_speed");
```

Comment:
I added this as a comment so we don't overlook this once we make the lightmapper work.